.prettierrc.js

```diff
@@ -3,4 +3,6 @@ module.exports = {
   tabWidth: 2,
   semi: true,
   singleQuote: true,
+  // Added this 
+  endOfLine: auto
```

Comment:
is "auto" not the default?

Reply:
Mine did not. Without it, there were a lot of errors

Reply:
good catch!